### PR TITLE
zh add translation content/zh/docs/tasks/administer-cluster/kubeadm/k…

### PR DESCRIPTION
**Pull request by tanjunchen**
…ubeadm-upgrade-1-14.md

>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.15 Features: set Milestone to 1.15 and Base Branch to dev-1.15
> 
> For pull requests on Chinese localization, set Base Branch to release-1.14
>
> For pull requests on Korean Localization: set Base Branch to dev-1.14-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by tanjunchen** [comment]
@xichengliudui  uh

File: content/zh/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md [hunk 1]
```diff
@@ -0,0 +1,556 @@
...
+
+<!-- - You need to have a kubeadm Kubernetes cluster running version 1.13.0 or later.
+- [Swap must be disabled](https://serverfault.com/questions/684771/best-way-to-disable-swap-in-linux).
+- The cluster should use a static control plane and etcd pods.
+- Make sure you read the [release notes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.14.md) carefully.
+- Make sure to back up any important components, such as app-level state stored in a database.
+  `kubeadm upgrade` does not touch your workloads, only components internal to Kubernetes, but backups are always a best practice. -->
+- 您需要有一个由 `kubeadm` 创建并运行着 1.13.0 或更高版本的 Kubernetes 集群。
+- [禁用 Swap](https://serverfault.com/questions/684771/best-way-to-disable-swap-in-linux)。
+- 集群应使用静态的控制平面和 etcd pods。
```
Comment (xichengliudui) [inline content/zh/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md]:
```suggestion
- 集群应使用静态的控制平面和 etcd pod。
```

File: content/zh/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md [hunk 2]
```diff
@@ -0,0 +1,556 @@
...
+- 务必仔细认真阅读 [发行说明](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.14.md)。
```
Comment (xichengliudui) [inline content/zh/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md]:
```suggestion
- 务必仔细认真阅读[发行说明](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.14.md)。
```

File: content/zh/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md [hunk 3]
```diff
@@ -0,0 +1,556 @@
...
+    [addons] Applied essential addon: CoreDNS
+    [addons] Applied essential addon: kube-proxy
+
+    [upgrade/successful] SUCCESS! Your cluster was upgraded to "v1.14.0". Enjoy!
+
+    [upgrade/kubelet] Now that your control plane is upgraded, please proceed with upgrading your kubelets if you haven't already done so.
+    ```
+
+<!-- 1.  Manually upgrade your CNI provider plugin. -->
+1.  手动升级你的 （CNI） 提供商插件。
```
Comment (xichengliudui) [inline content/zh/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md]:
```suggestion
1.  手动升级你的 CNI 供应商插件。
```

File: content/zh/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md [hunk 4]
```diff
@@ -0,0 +1,556 @@
...
+
+    <!-- Your Container Network Interface (CNI) provider may have its own upgrade instructions to follow.
+    Check the [addons](/docs/concepts/cluster-administration/addons/) page to
+    find your CNI provider and see whether additional upgrade steps are required. -->
+    您的容器网络接口（CNI）应该提供了程序自身的升级说明。
+    检查 [addons](/docs/concepts/cluster-administration/addons/) 页面以查找您 CNI 所提供的程序，并查看是否需要其他升级步骤。
```
Comment (xichengliudui) [inline content/zh/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md]:
```suggestion
    检查[插件](/docs/concepts/cluster-administration/addons/)页面查找您 CNI 所提供的程序，并查看是否需要其他升级步骤。
```

File: content/zh/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md [hunk 5]
```diff
@@ -0,0 +1,556 @@
...
+- 应用新的 `kube-dns` 和 `kube-proxy` 清单，并强制创建所有必需的 RBAC 规则。
+- 如果旧文件在 180 天后过期，将创建 API 服务器的新证书和密钥文件并备份旧文件。
+
+<!-- `kubeadm upgrade node experimental-control-plane` does the following on additional control plane nodes:
+- Fetches the kubeadm `ClusterConfiguration` from the cluster.
+- Optionally backups the kube-apiserver certificate.
+- Upgrades the static Pod manifests for the control plane components. -->
+`kubeadm upgrade node experimental-control-plane` 实验控制平面在其他控制平面节点上执行以下操作：
+- 从集群中获取 kubeadm `ClusterConfiguration`。
+- 可选地备份 kube-apiserver 书。
```
Comment (xichengliudui) [inline content/zh/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md]:
```suggestion
- 可选地备份 kube-apiserver 证书。
```

File: content/zh/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md [hunk 6]
```diff
@@ -0,0 +1,556 @@
...
+- Checks that your cluster is in an upgradeable state:
+  - The API server is reachable
+  - All nodes are in the `Ready` state
+  - The control plane is healthy
+- Enforces the version skew policies.
+- Makes sure the control plane images are available or available to pull to the machine.
+- Upgrades the control plane components or rollbacks if any of them fails to come up.
+- Applies the new `kube-dns` and `kube-proxy` manifests and makes sure that all necessary RBAC rules are created.
+- Creates new certificate and key files of the API server and backs up old files if they're about to expire in 180 days. -->
+## 它是怎么运行的
```
Comment (xichengliudui) [inline content/zh/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md]:
```suggestion
## 它是怎么工作的
```

**Comment by xichengliudui** [comment]
Can you tell me the preview link?

**Comment by tanjunchen** [comment]
https://deploy-preview-16406--k8s-v1-14.netlify.com/zh/docs/tasks/administer-cluster/kubeadm/

**Comment by tanjunchen** [comment]
@xichengliudui  https://deploy-preview-16406--k8s-v1-14.netlify.com/zh/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14/

**Comment by xichengliudui** [comment]
还需要更新一下，预览不对

**Comment by xichengliudui** [comment]
image
一共三个代码块，将代码块中的注释取出，放到代码块上方或下方。

**Comment by tanjunchen** [comment]
OK Thanks !!!

**Pull request merged**